hut-finder-api/cmd/server/main.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright © 2024 Yunu Cho yunu121@gmail.com, Jake Dalton cqsmico7@gmail.com
+*/
+
+package main
+
+import (
+	"hut-finder-api/pkg/api"
+	"hut-finder-api/pkg/config"
+	"hut-finder-api/pkg/repository"
+	"log"
+	"os"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+	err := config.LoadConfig()
+	if err != nil {
+		log.Fatalf("Could not load config: %v", err)
+	}
```

Comment:
Is it possible for something more graceful than a fatal error if .env is not present, some fallback or defaults maybe?
Some docs for setting up .env would also be good.